### ISSUE TITLE
Add stylelint-disable-next-line disableRanges command

### ISSUE THREAD

**Issue by giladgray** [stylelint/stylelint#1614]
Would be super cool to add a `stylelint-disable-next-line` flag a la [TSLint](https://github.com/palantir/tslint#rule-flags) so my comments could always live on their own line.

**current syntax:**

```scss
.selector {
  // this color is essential
  color: blue !important; // stylelint-disable-line declaration-no-important
}

.selector2 {
  // this color is essential
  // stylelint-disable declaration-no-important
  color: blue !important; 
  // stylelint-enable declaration-no-important
  background: etc...;
}
```

**proposed syntax:**

```scss
.selector {
  // this color is essential
  // stylelint-disable-next-line declaration-no-important
  color: blue !important; 
  background: etc...;
}
```

**Comment by jeddy3** [comment]
This seems like a valid feature request.

Does anyone have any objections to accepting a (well tested) PR for it?

**Comment by ntwb** [comment]
SGTM

**Comment by davidtheclark** [comment]
No objections here.

**Comment by jeddy3** [comment]
@giladgray, as in https://github.com/stylelint/stylelint/issues/1615, feel free to contribute this feature. You'll the find the `disable-ranges` code [here](https://github.com/stylelint/stylelint/blob/master/src/disableRanges.js) and the tests [here](https://github.com/stylelint/stylelint/blob/master/src/__tests__/disableRanges-test.js) and [here](https://github.com/stylelint/stylelint/blob/master/src/__tests__/disableRanges-integration-test.js). The Developer Guide [explains](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#running-tests) how to run the tests.

**Comment by giladgray** [comment]
🎆  thanks @davidtheclark! much appreciated.
